### ISSUE TITLE
Support for hackintos

### ISSUE THREAD

**Issue by jrsarath** [BrianLima/HueHue#17]
Any intention to build one for Mac osx

**Comment by BrianLima** [comment]
I don't think it would be possible for the current app. I might write an alternative version with basic control for just Arduinos, but it is not possible to port normal .net WPF desktop apps to multi platform. Neither the device specific DLLs

**Comment by jrsarath** [comment]
Nevermind, you did a great job, 
continue with this one.. 
i saw a bug in this release sometime it closes it-self while playing games

**Comment by BrianLima** [comment]
This becomes relevant again since .Net WPF is moving open-source and multi-plat.

I plan to revisit this when more information about WPF apps on MacOS become available.

**Comment by jrsarath** [comment]
That would be super cool if you come across to support macOS

**Comment by SamPoulton** [comment]
What about [Wine](https://www.winehq.org/)?
